components/AppHeader2/wrapper.tsx

```diff
@@ -115,6 +115,18 @@ export default function AppHeader2_Wrapper() {
           {user ? 'Sign Out' : 'Sign In'}
         </button>
       </div>
+      <button
+        className="md:hidden mt-2 absolute right-[2rem] lg:right-[10rem] py-3 px-5 rounded-[30px] bg-[#40B7BA] font-bold text-white ml-3 border-2 border-white"
+        onClick={async () => {
+          if (user) {
+            await signOut();
+          } else {
+            await router.push('/auth');
+          }
+        }}
+      >
+        {user ? 'Sign Out' : 'Sign In'}
+      </button>
```

Comment:
nit: might wanna move the button a little bit on mobile view. 
![Shot-2024-08-21-151624](https://github.com/user-attachments/assets/311f51a7-941c-4fed-8245-09edd3fd9dec)

Reply:
(i have not fixed this yet)